src/lib/subtract.js

```diff
@@ -1,4 +1,6 @@
-const subtract = (predicate, array) => {
-
+const subtract = (num1,num2) => {
+  var difference = num1 - num2;
+  console.log(difference);
```

Comment:
remove `console.log`, there's no reason you should be logging output for this method.